xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -597,7 +597,7 @@ void CGUIDialogVideoInfo::Play(bool resume)
     return;
   }
 
-  CFileItem movie(*m_movieItem->GetVideoInfoTag());
+  CFileItem movie(*m_movieItem);
```

Comment:
There are overloaded constructors, the one that takes a VideoInfoTag does not populate all the member level isFolder type data it uses some internal logic to set them from data in the VideoInfoTag which is not consistent with what the original item has.
Using the base Item it allows the new movie item to be populated using the full item info.
I feel it is actually more correct to use the constructor that passes in the full item if it is available, in this instance the full item is available and should be used.